pydatalab/src/pydatalab/routes/v0_1/info.py

```diff
@@ -143,3 +144,36 @@ def list_block_types():
             ).json()
         )
     )
+
+
+def get_all_models():
+    return Item.__subclasses__()
+
+
+@INFO.route("/info/types", methods=["GET"])
+def list_supported_types_schemas():
+    """Returns a dictionary of supported item types and their schemas."""
+    schemas = {cls.__name__.lower(): cls.schema() for cls in get_all_models()}
```

Comment:
Very minor, but I think I would use the item's `type` here as it should be the ground truth. This might need a bit of manipulation, but I think you can get it from e.g. `cls.schema()["properties"]["type"]["default"]` or something similar

Reply:
also I think here I would actually just list the types themselves, rather than providing their full schema. Then each schema can be individually resolved under `/info/types/<x>`